apps/wallet-mobile/src/features/Settings/WalletSettings/WalletSettingsScreen.tsx

```diff
@@ -147,26 +139,16 @@ export const WalletSettingsScreen = () => {
   )
 }
 
-const getNetworkName = (networkId: NetworkId) => {
-  // note(v-almonacid): this throws when switching wallet
-  try {
-    const config = getNetworkConfigById(networkId)
-    return config.MARKETING_NAME
-  } catch (_e) {
-    return '-'
-  }
-}
-
-const getWalletType = (implementationId: WalletImplementationId): MessageDescriptor => {
-  if (isByron(implementationId)) return messages.byronWallet
-  if (isHaskellShelley(implementationId)) return messages.shelleyWallet
+const getWalletType = (implementation: Wallet.Implementation): MessageDescriptor => {
+  if (implementation === 'cardano-bip44') return messages.byronWallet
+  if (implementation === 'cardano-cip1852') return messages.shelleyWallet
```

Comment:
could `isShelley` and `isByron` be used here?

Reply:
I dropped, mainly because it was misleading, `isByron` means bip44, isShelley (15/24) means cip1852. for someone that is not familiar with Byron/Shelley but knows BIPs, it is simpler to understand.